examples/manifold/plot_lle_digits.py

```diff
@@ -131,7 +131,7 @@ def plot_embedding(X, title, ax):
     "LTSA LLE embedding": LocallyLinearEmbedding(
         n_neighbors=n_neighbors, n_components=2, method="ltsa"
     ),
-    "MDS embedding": MDS(n_components=2, n_init=1, max_iter=100),
+    "MDS embedding": MDS(n_components=2, n_init=1, max_iter=100, n_jobs=-1),
```

Comment:
Have you tried changing `max_iter`, here?

Reply:
`max_iter ` bellow 100 is not converging  and there's room to go to 125.
`n_jobs ` is helping reduce runtime for MDS. for n_jobs=2 it is about 0.25 percent.
with `max_iter `=125 and `n_jobs`=2 it takes 3.9 +/- 0.2  seconds.